meson-scripts/stress_tests.ini

```diff
@@ -1,54 +1,54 @@
 [scx_lavd]
 sched: scx_lavd
 sched_args: -v
-stress_cmd: stress-ng -t 14 --aggressive -M -c `nproc` -f `nproc`
-timeout_sec: 15
+stress_cmd: stress-ng -t 31 --aggressive -M -c `nproc` -f `nproc` --affinity 1 --affinity-sleep 1 --affinity-pin
```

Comment:
If we want to stress the scheduler even more we could use `--affinity-delay 1`, since it'd spin on `sched_yield()` instead of using `nanosleep()`.

Basically with such a short sleep time, `nanosleep()` is likely handled in the VDSO, so the task remains in user-space and once it gets the CPU it'd run for the entire time slice. With `sched_yield()` as soon the task gets the CPU it gives it back and gets queued again in the scheduler, so we should have more `ops.enqueue()` invocations and more chances to expose stall conditions.